bitnami/elasticsearch/8/debian-12/prebuildfs/opt/bitnami/scripts/libnet.sh

```diff
@@ -68,7 +68,13 @@ get_machine_ip() {
         error "Could not find any IP address associated to hostname ${hostname}"
         exit 1
     fi
-    echo "${ip_addresses[0]}"
+
+    # Check if the first IP address is IPv6 and add brackets
+    if [[ "${ip_addresses[0]}" =~ : ]]; then
```

Comment:
You could try `validate_ipv6` function by loading `. /opt/bitnami/scripts/libvalidations.sh` at the beginning of the file:

```suggestion
    if validate_ipv6 "${ip_addresses[0]}"; then
```